documentation/asciidoc/computers/config_txt/codeclicence.adoc

```diff
@@ -4,7 +4,7 @@ Hardware decoding of additional codecs on the Raspberry Pi 3 and earlier models
 
 The Raspberry Pi 4 has permanently disabled hardware decoders for MPEG2 and VC1. These codecs cannot be enabled, so a hardware codec licence key is not needed. Software decoding of MPEG2 and VC1 files performs well enough for typical use cases.
 
-The Raspberry Pi 5 has HEVC and H.265 hardware decoders. Both are enabled by default and do not require a licence key, so a hardware codec licence key is not needed.
+The Raspberry Pi 5 has HEVC/H.265 hardware decoding. This decoding is enabled by default, so a hardware codec licence key is not needed.
```

Comment:
I think that's generally written as HEVC (H.265). But YMMV.

Reply:
I had no clue at all that H.264 was also known as AVC, so there's that! 🤣

Reply:
```suggestion
The Raspberry Pi 5 has H.265 (HEVC) hardware decoding. This decoding is enabled by default, so a hardware codec licence key is not needed.
```